skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,137 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+This module is designed to facilitate image arithmetic and logical operations
+on image data sets. The included functions supplement the logical operations
+currently provided in numpy in order to provide a complete set of logical
+operations for data analysis.
+
+The new functions include:
+    logical_nand: Identifies all elements NOT included in BOTH inputs
+    logical_nor: Identifies all elements NOT included in EITHER input
+    logical_sub: Identifies all elements ONLY included in input_1
+"""
+
+import numpy as np
+from numpy import (logical_and, logical_or, logical_not, logical_xor, add,
+                   subtract, multiply, divide)
+
+def logical_nand(x1,
+                 x2):
+    """
+    This function enables the computation of the LOGICAL_NAND of two image or 

```

Comment:
This should be terser.

```python
Computes the truth value of NOT (x1 AND x2) element wise.
```

The longer narrative should go in the rst-based prose documentation.